src/App.js

```diff
@@ -1,29 +1,8 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Kindly ensure that your `<App />` component is written as a class-based component according to the project requirement that all component must be class-based 👍🏽